tests/wpunit/Admin/AdminTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Includes tests for the Admin.
+ */
+
+namespace Admin;
+
+use WPBDP__CPT_Integration;
+use WPBDP_Themes;
+use WPBDP_Admin;
+use WPBDP\Tests\WPUnitTestCase;
+use WPBDP;
+use Codeception\Util\Debug;
+
+/**
+ * Tests for the Admin.
+ *
+ * @since x.x
+ */
+class AdminTest extends WPUnitTestCase {
+
+	/**
+	 * WpunitTester instance.
+	 *
+	 * @var \WpunitTester
+	 */
+	protected $tester;
+
+	/**
+	 * WPBDP_Admin instance.
+	 *
+	 * @var \WPBDP_Admin
+	 */
+	private static $admin;
+
+	/**
+	 * Admin user ID.
+	 *
+	 * @var int
+	 */
+	protected static $admin_user_id;
+
+	/**
+	 * Editor user ID.
+	 *
+	 * @var int
+	 */
+	protected static $editor_user_id;
+
+	/**
+	 * WPBDP menu ID.
+	 *
+	 * @var string
+	 */
+	protected static $menu_id;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		static::$admin_user_id = $factory->user->create( [ 'role' => 'administrator' ] );
+		static::$editor_user_id = $factory->user->create( [ 'role' => 'editor' ] );
+
+		if ( ! class_exists( 'WPBDP_Admin' ) ) {
+			require_once WPBDP_INC . 'admin/class-admin.php';
+		}
+
+		static::$admin = new WPBDP_Admin();
+		static::$menu_id = static::$admin->get_menu_id();
+	}
+
+	public function testMenuExistsAsAdmin() {
+		$this->tester->wantToTest( 'Admin Menu Exists As Admin' );
+		$this->wpbdp_admin_menu_exists( static::$admin_user_id );
+	}
+
+	public function testMenuExistsAsEditor() {
+		$this->tester->wantToTest( 'Admin Menu Exists For Editor User' );
+		$this->wpbdp_admin_menu_exists( static::$editor_user_id );
+	}
+
+	public function testListingPostTypeDoesNotExistAtTopLevelAsAdmin() {
+		$this->tester->wantToTest( 'Listing Post Type Does Not Exist At Top Level As Admin' );
+		$this->wpbdp_listing_post_type_at_top_level_exists( static::$admin_user_id );
+	}
+
+	public function testCombineListingPostTypeAndWPBDP_Admin() {
```

Comment:
@stephywells I'll complete this test after finding a way to check Listing Post Type as a top-level menu.